src/subcommand/wallet/batch_command.rs

```diff
@@ -17,12 +17,12 @@ impl Batch {
 
     let batchfile = batch::File::load(&self.batch)?;
 
-    let parent_info = wallet.get_parent_info(batchfile.parent)?;
+    let parents_info = wallet.get_parents_info(batchfile.parents.clone())?;
```

Comment:
I would still just call this `parent_info`. `parents_info` sounds awkward, and since it's a vec, the fact that there is more than one is encoded in the type.